code/game/machinery/porta_turret/portable_turret.dm

```diff
@@ -917,3 +917,129 @@
 				on = 0
 				spawn(100)
 					on = 1
+
+/////// MANNED TURRET ////////
+
+/obj/machinery/manned_turret
+	name = "machine gun turret"
+	desc = "A mounted gun that, while antiquated, has been a black market staple for centuries."
+	icon = 'icons/obj/turrets.dmi'
+	icon_state = "protoemitter"
+	can_buckle = TRUE
+	density = TRUE
+	buckle_lying = 0
+	layer = 4.2
+	var/view_range = 10
+	var/cooldown = 0
+	var/projectile_type = /obj/item/projectile/bullet/weakbullet3
+
+//BUCKLE HOOKS
+
+/obj/machinery/manned_turret/unbuckle_mob(mob/living/buckled_mob,force = 0)
+	playsound(src,'sound/mecha/mechmove01.ogg', 50, 1)
+	for(var/obj/item/I in buckled_mob.held_items)
+		if(istype(I, /obj/item/weapon/turret_control))
+			qdel(I)
+	if(istype(buckled_mob))
+		buckled_mob.pixel_x = 0
+		buckled_mob.pixel_y = 0
+		if(buckled_mob.client)
+			buckled_mob.client.change_view(world.view)

```

Comment:
```reset_perspective()``` is the wrong proc for this, that's to do with client eye positioning.
```change_view()``` is correct.